client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/TransferInvestigationsDialogs/TransferInvestigationInvestigator.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Autocomplete } from '@material-ui/lab';
+import { yupResolver } from '@hookform/resolvers';
+import { TextField, Typography } from '@material-ui/core';
+import { useForm, FormProvider, Controller } from 'react-hook-form';
+
+import InvestigatorOption from 'models/InvestigatorOption';
+
+import useStyles from './TransferDialogsStyles';
+import validationSchema from './TransferInvestigationInvestigatorSchema';
+import TransferInvestigationDialog from './TransferInvestigationDialog';
+import TransferInvestigationInvestigatorInputNames from './TransferInvestigationInvestigatorInputNames';
+
+const transferInvestigatorFormName = 'transferInvestigator';
+
+const TransferInvestigationDesk = (props: Props) => {
+
+    const { allInvestigators, open, onClose, onConfirm } = props;
+
+    const classes = useStyles();
+    
+    const methods = useForm({
+        mode: 'all',
+        resolver: yupResolver(validationSchema),
+        defaultValues: {
+            [TransferInvestigationInvestigatorInputNames.INVESTIGATOR]: null,
```

Comment:
Maybe change the default value to an empty InvestigatorOption object?

Reply:
because react hook form sucks 😂